packages/site-parsers/src/parsers/wix/components/html.js

```diff
@@ -0,0 +1,20 @@
+const axios = require( 'axios' );
+const { createBlock } = require( '@wordpress/blocks' );
+
+const SUPPORTED_SOURCE = [ 'htmlEmbedded' ];
+
+module.exports = {
+	type: 'HtmlComponent',
+	parseComponent: async ( component, { metaData } ) => {
+		if ( SUPPORTED_SOURCE.indexOf( component.dataQuery.sourceType ) === -1 )
+			return null;
+		const htmlContentUrl =
+			metaData.serviceTopology.staticHTMLComponentUrl +
+			component.dataQuery.url;
+
+		return await axios
```

Comment:
Is there a particular reason for using axios here, rather than `window.fetch()`?

Reply:
Since the window fetch is not available in the node.js, I was between two options:
- native window.fetch polyfill
- or Axios

I have a good experience with the Axios, and that's the only why. If you have better lib, we switch it easily.